project/scripts/winCmdTests.bat

```diff
@@ -14,10 +14,10 @@ if not %ERRORLEVEL%==0 endlocal& exit /b 1
 call "%_PREFIX%\bin\scalac.bat" -d "%_OUT_DIR%" "%_SOURCE%"
 if not %ERRORLEVEL%==0 endlocal& exit /b 1
 
-call "%_PREFIX%\bin\scala.bat" -classpath "%_OUT_DIR%" HelloWorld
+call "%_PREFIX%\bin\scala.bat" --power -classpath "%_OUT_DIR%" -M HelloWorld --offline --server=false
 if not %ERRORLEVEL%==0 endlocal& exit /b 1
 
-call "%_PREFIX%\bin\scala.bat" -classpath "%_OUT_DIR%" -J-Xmx512m HelloWorld
+call "%_PREFIX%\bin\scala.bat" --power -classpath "%_OUT_DIR%" -J -Xmx512m -M HelloWorld --offline --server=false
 if not %ERRORLEVEL%==0 endlocal& exit /b 1
```

Comment:
Clearing `scala-cli` files and folders too here

Reply:
same thing here, we are only running a class, so no files are created